src/ui/map/TileSet.svelte

```diff
@@ -1,5 +1,6 @@
 <script>
   import { getContext, setContext } from "svelte";
+  import config from "../../config";
```

Comment:
If config is not used , probably we can remove the import.

Reply:
My bad missed that, yep will remove it.